.vsts-pipelines/templates/phases/default-build.yml

```diff
@@ -70,6 +70,7 @@ phases:
     demands: ${{ parameters.demands }}
   variables:
     AgentOsName: ${{ parameters.agentOs }}
+    ASPNETCORE_TEST_LOG_DIR: ${{ parameters.artifacts.path }}/testlogs
```

Comment:
Can we put this in artifacts/logs/ instead? We already put other logging files into this folder.

Reply:
If I understand correctly the publish task uses `pathtoPublish: ${{ parameters.artifacts.path }}` to determine what to publish as an artifact. So artifacts/logs wouldn't be picked up

Reply:
We can change artifact publishing to include other folders. I think using artifacts/build/testlogs/ may impact the way the "known artifacts" check works. KoreBuild scans artifacts/build/testlogs/ to make sure everything in it is accounted for.